pixl_dcmd/tests/conftest.py

```diff
@@ -126,67 +126,6 @@ def row_for_testing_image_with_pseudo_patient_id(
     return db_session
 
 
-def ids_for_parameterised_test(val: pathlib.Path) -> str:
-    """Generate test ID for parameterised tests"""
-    return str(val.stem)
-
-
-@pytest.fixture()
-@pytest.mark.parametrize(
-    ("yaml_file"), PROJECT_CONFIGS_DIR.glob("*.yaml"), ids=ids_for_parameterised_test
-)
-def row_for_dicom_testing(db_session, yaml_file) -> Session:
```

Comment:
this is not needed (any tests that need the DB already use the `rows_in_session` fixture instead)